### PR TITLE
docs: Revert all changes after the license change

### PR DESCRIPTION
**Pull request by mdjermanovic**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

Refs https://github.com/eslint/eslint/issues/17225, fixes step 2.

#### What changes did you make? (Give an overview)

Checked out https://github.com/eslint/eslint/commit/b3634f695ddab6a82c0a9b1d8695e62b60d23366 and copied its `docs/` folder over the current one.

#### Is there anything you'd like reviewers to focus on?

This reverts all docs changes after https://github.com/eslint/eslint/commit/b3634f695ddab6a82c0a9b1d8695e62b60d23366, including the bot's and team members' commits. Is that okay?

Also, this reverts only docs parts of the commits. For example, https://github.com/eslint/eslint/commit/880a4317b949e575a4a6c5e8baaba1eea7674cc6 had changes in both code and docs. Its code changes will stay, while the docs changes will be removed. Is that okay?

### CODE REVIEW

**Comment by amareshsm** [comment]
> Also, this reverts only docs parts of the commits. For example, https://github.com/eslint/eslint/commit/880a4317b949e575a4a6c5e8baaba1eea7674cc6 had changes in both code and docs. Its code changes will stay, while the docs changes will be removed. Is that okay?

After reverting this PR are we immediately going to resubmit the removed documentation?

File: docs/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "docs-eslint",
     "private": true,
-    "version": "8.41.0",
+    "version": "8.36.0",
```
Comment (nzakas) [inline docs/package.json:4]: I think we probably want to keep this at 8.41.0 to match the ESLint version?

File: docs/src/_data/sites/en.yml [hunk 2]
```diff
@@ -85,7 +85,6 @@ footer:
     description: Selecting a language will take you to the ESLint website in that language.
     change_language: Change Language
     language: Language
-    latest: Latest
```
Comment (nzakas) [inline docs/src/_data/sites/en.yml:88]: I think we need to keep this?

File: docs/src/_data/sites/zh-hans.yml [hunk 3]
```diff
@@ -83,7 +83,6 @@ footer:
     description: 切换到你所选择语言版本对应的 ESLint 网站。
     change_language: 更改语言
     language: 语言
-    latest: 最新
```
Comment (nzakas) [inline docs/src/_data/sites/zh-hans.yml:86]: Ditto

File: docs/src/_includes/components/language-switcher.html
Comment (nzakas) [inline docs/src/_includes/components/language-switcher.html:1]: Safe to keep

File: docs/src/_includes/layouts/base.html
Comment (nzakas) [inline docs/src/_includes/layouts/base.html:1]: Safe to keep

File: docs/src/_includes/layouts/doc.html
Comment (nzakas) [inline docs/src/_includes/layouts/doc.html:1]: Safe to keep

File: docs/src/assets/js/scroll-up-btn.js
Comment (nzakas) [inline docs/src/assets/js/scroll-up-btn.js:1]: Safe to Keep

File: docs/src/assets/scss/components/alert.scss
Comment (nzakas) [inline docs/src/assets/scss/components/alert.scss:1]: Safe to keep

**Review by nzakas — CHANGES_REQUESTED** [review]
Thanks for going through and doing this.

I started leaving notes on each file, but basically we just need to revert the *.md files, as those are the only ones that contain documentation. All of the other file types are covered under the MIT license only and we should leave as-is.

**Review by aladdin-add — APPROVED** [review]
LGTM, thanks! 👍

**Review by nzakas — APPROVED** [review]
LGTM. Just leaving open so we can coordinate how we want to merge this in.

**Pull request merged**